### ISSUE TITLE
Staging Review finding: Accredited Representative Portal wordmark is mislabeled for screen reader users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104049]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Rep Facing (ARF) + Accredited Rep Mgmt (ARM)
**Product:** ARF (Accredited Rep Portal MVP) + ARM (Appoint a Rep 2.0)
**Feature:** Form 21-22: Digital Submission (ARM) and Accept/Decline (ARF)

## Findings details
**VA.gov Experience Standard - issue:** [User can identify an element and its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/representative](https://staging.va.gov/representative) and elsewhere in the portal, in the heading the wordmark logo "VA Accredited Represenative Portal" is announced to screen reader users as "Veteran Affairs logo." This isn't what the element is, and doesn't provide a clear indication of where you go if you follow the link.

<details><summary>Link, screenshot or steps to recreate</summary>

Currently coded as:
```
<a data-testid="nav-home-link" aria-label="VA logo" class="nav__link vads-u-display--flex" href="/representative">
   <img data-testid="mobile-logo" class="nav__logo mobile" src="/img/va.svg" alt="Veteran Affairs logo">
   <span class="nav__logo-text mobile">Accredited Representative Portal</span>
   <img data-testid="desktop-logo" class="nav__logo nav__logo--desktop desktop" src="/img/arp-header-logo-dark.svg" alt="VA Accredited Representative Portal Logo, U.S. Department of Veterans Affairs"></a>
```

</details>

## Recommended action

There are a few things attempting to give a screen reader-friendly name to this element, but the one that wins is the `aria-label` on the link wrapping the whole thing. Update that to something like "VA Accredited Representative Portal" and you're good.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jquispe-oddball** [comment]
submitted pr 
https://github.com/department-of-veterans-affairs/vets-website/pull/35023

**Comment by jquispe-oddball** [comment]
This is on staging, putting a screenshot of a recording below. Ticket can be closed @candilemoine 

<img width="666" alt="Image" src="https://github.com/user-attachments/assets/f3974f10-0c13-48f8-8dcb-17f06313366d" />
